src/v2/records_class.py

```diff
@@ -0,0 +1,207 @@
+            row['Year'], row['Genres'], row['Num Votes'], row['Release Date'], row['Directors']]
+            self.cursor.execute(columns, values)
+            self.conn.commit()
+        
+        except (Exception, ps.DatabaseError) as error:
+            print(error)
+            return -1
+
+    def read(self, query, *params):
+        '''
+        Summary Line 
+        Extended Description of read(self, query, params)
+
+        parameters:
+        ----------
+        query 
+        params
+
+        executes sql query along with params if exists
+        prints the status after each execution
+
+        raises exception if any
+        '''
+        try:
+            values =[*params]
+            self.cursor.execute(query, values)
+            result=self.cursor.fetchall()
+            print(result)
+            print("\n")
+        except (Exception, ps.DatabaseError) as error:
+            print(error)
+            return -1
+        
+    def update(self, const, new_rating):
+        '''
+        Summary Line
+        Extended description of update(self, const, new_rating)
+
+        parameters:
+        -----------
+        const //primary key
+        new_rating
+
+        selects record in ratings table that matches const
+
+        checks the validity of the new_rating 
+        proceeds with updation only if values ranging between 0 and 10
+        
+        updates the value of column your_rating with new_rating
+        also updates date_rated with current date
+        prints the status
+
+        raises exception if any
+        '''
+        try: 
+            if (new_rating>=0.0 and new_rating<=10.0):
+
+                day=datetime.now()
+                day=day.strftime("%d/%m/%Y")
+                try:
+                    query = '''UPDATE ratings SET Your_Rating = %s, Date_Rated = %s
+                    WHERE Const = %s;'''
+                    values=[new_rating, day, const]
+                    self.cursor.execute(query, values)    
+                    self.conn.commit()
+                    print("values updated in the database successfully...\n")
+
+                except (Exception, ps.DatabaseError) as error:
+                    print(error)
+                    return -1
+            else:
+                print("enter a valid value between 0 and 10!!\n")
+                return -1
+        except:
+            print("enter a valid value between 0 and 10!!\n")
+            return -1
+
+    def delete(self, const):
+        '''
+        Summary Line 
+        Extended Description of delete(self, const)
+
+        parameters:
+        -----------
+        const //primary key
+
+        selects the rating in the record that matches with const
+        sets the rating to 0
+        prints the status
+
+        raises exception if any
+        '''
+        try:
+            query='''UPDATE ratings SET Your_Rating = NULL WHERE Const = %s;'''
```

Comment:
this is an update query. use delete statement. this is doing something very different from expected behaviour

Reply:
you have to delete the record from table